docs/demos/telecom-customer-churn-airflow/telecom-customer-churn-airflow.md

```diff
@@ -0,0 +1,302 @@
+In the end, you will have a pipeline that:
+
+- Loads some data
+- Trains two different models
+- Evaluates which model is best
+- Saves that model to S3
+
+!!!hint
+    You can expand on this demo by loading the pushed model into [MLFlow](/tools-and-applications/mlflow/mlflow), or automatically deploying it into some application, like in the [Credit Card Fraud Demo](/demos/credit-card-fraud-detection-mlflow/credit-card-fraud)
+
+The models we build are used to predict customer churn for a Telecom company using structured data. The data contains fields such as: If they are a senior citizen, if they are a partner, their tenure, etc.
+
+## Deploying the demo
+
+### Pre-requisites
+
+- Fork this git repository into a GitHub or GitLab repo (the demo shows steps for GitHub, but either works): [https://github.com/red-hat-data-services/telecom-customer-churn-airflow](https://github.com/red-hat-data-services/telecom-customer-churn-airflow)
+- Have [Airflow](/tools-and-applications/airflow/airflow/) running in a cluster and point Airflow to the cloned git repository.
+- Have access to some S3 storage (this guide uses ODF with a bucket created in the namespace "airflow").
+- Have [Red Hat OpenShift Data Science](/getting-started/openshift-data-science/) (RHODS) running in a cluster. Make sure you have admin access in RHODS, or know someone who does.
+
+!!! note
+    Note: You can use [Open Data Hub](/getting-started/opendatahub/) instead of RHODS, but some instructions and screenshots may not apply
+
+### 1: Open up Airflow
+
+You find the route to the Airflow console through this command: `oc get route -n airflow`
+
+![Airflow Route](img/Airflow_Route.png)
+
+Enter it in the browser and you will see something like this:
+
+![Airflow Main Hub](img/Airflow_main_hub.png)
+
+Keep that open in a tab as we will come back to Airflow later on.
+
+### 2: Add Elyra as a Custom Notebook Image
+
+It's possible to build pipelines by creating an Airflow DAG script in python. Another, arguably simpler, method is to use Elyra to visually build out the pipeline and then submit it to Airflow.  
+Most of this demo is going to be revolving around using Elyra together with Airflow, but at the very end, there will be a bonus section for how to use Airflow independently.
+
+To get access to Elyra, we will simply import it as a custom notebook image.  
+Start by opening up RHODS by clicking on the 9-square symbol in the top menu and choosing "Red Hat OpenShift Data Science".
+
+![Open RHODS](img/Open_RHODS.png)
+
+Then go to Settings -> Notebook Images and press "Import new image".  
+If you can't see Settings then you are lacking sufficient access. Ask your admin to add this image instead.
+
+![Import Notebook Image](img/Import_Notebook_Image.png)
+
+Under Repository enter: `quay.io/eformat/elyra-base:0.2.1` and then name it something like `Elyra`.
```

Comment:
Can you try to switch to quay.io/opendatahub-contrib/workbench-images:jupyter-datascience-ubi9-py39_2023b_latest
This is the newest Elyra+DataScience custom image, which is the base for the new Datascience image that will be released shortly in ODH/RHODS, integrating Elyra from the start.

Reply:
Plus it comes with pre-built runtime images that are compatible with OpenShift (contrary to base Elyra).